Debian policy.
This patch updates default plugin path, game db path.
It also updates shaders data path.
Index: pcsx2.snapshot-4292/pcsx2/gui/AppConfig.cpp
===================================================================
--- pcsx2.snapshot-4292.orig/pcsx2/gui/AppConfig.cpp
+++ pcsx2.snapshot-4292/pcsx2/gui/AppConfig.cpp
@@ -164,7 +164,8 @@
 
 	wxDirName GetPlugins()
 	{
-		return AppRoot() + Base::Plugins();
+		// return AppRoot() + Base::Plugins();
+		return wxDirName( L"/usr/lib/games/pcsx2" ) + Base::Plugins();
 	}
 
 	wxDirName GetThemes()
Index: pcsx2.snapshot-4292/plugins/zzogl-pg/opengl/ZZoglCreate.cpp
===================================================================
--- pcsx2.snapshot-4292.orig/plugins/zzogl-pg/opengl/ZZoglCreate.cpp
+++ pcsx2.snapshot-4292/plugins/zzogl-pg/opengl/ZZoglCreate.cpp
@@ -291,18 +291,23 @@
 
 __forceinline bool LoadShadersFromDat()
 {
-	FILE* fres = fopen("ps2hw.dat", "rb");
 
+    // First try an absolute location for linux distribution
+    FILE* fres = fopen("/usr/share/games/pcsx2/shaders/ps2hw.dat", "rb");
 	if (fres == NULL)
-	{
-		fres = fopen("plugins/ps2hw.dat", "rb");
+    {
+        fres = fopen("ps2hw.dat", "rb");
+        if (fres == NULL)
+        {
+            fres = fopen("plugins/ps2hw.dat", "rb");
 
-		if (fres == NULL)
-		{
-			ZZLog::Error_Log("Cannot find ps2hw.dat in working directory. Exiting.");
-			return false;
-		}
-	}
+            if (fres == NULL)
+            {
+                ZZLog::Error_Log("Cannot find ps2hw.dat in working directory. Exiting.");
+                return false;
+            }
+        }
+    }
 
 	fseek(fres, 0, SEEK_END);
 
Index: pcsx2.snapshot-4292/pcsx2/gui/AppGameDatabase.h
===================================================================
--- pcsx2.snapshot-4292.orig/pcsx2/gui/AppGameDatabase.h
+++ pcsx2.snapshot-4292/pcsx2/gui/AppGameDatabase.h
@@ -51,8 +51,8 @@
 		Console.WriteLn( "(GameDB) Unloading..." );
 	}
 
-	AppGameDatabase& LoadFromFile(const wxString& file = L"GameIndex.dbf", const wxString& key = L"Serial" );
-	void SaveToFile(const wxString& file = L"GameIndex.dbf");
+	AppGameDatabase& LoadFromFile(const wxString& file = L"/var/games/pcsx2/GameIndex.dbf", const wxString& key = L"Serial" );
+	void SaveToFile(const wxString& file = L"/var/games/pcsx2/GameIndex.dbf");
 };
 
 static wxString compatToStringWX(int compat) {
